src/processor/operator/persistent/reader/csv/driver.cpp

```diff
@@ -340,6 +341,9 @@ void copyStringToVector(ValueVector* vector, uint64_t rowToAdd, std::string_view
             vector, rowToAdd, reinterpret_cast<char*>(blobBuffer.get()), blobLen);
     } break;
     case LogicalTypeID::STRING: {
+        if (!utf8proc::Utf8Proc::isValid(strVal.data(), strVal.length())) {
+            throw common::CopyException{"Invalid UTF8-encoded string."};
```

Comment:
Can we also print the strVal in error message?

Reply:
I think printing the invalid utf8 characters is undefined behaviour and causes problems when writing the exception test for utf8 strings.